### DISCUSSION TITLE
Webhook is not working properly

### DISCUSSION THREAD

**Discussion by a-akashsoni** [Kyon147/laravel-shopify#131] [Q&A]
I have created a webhook for order creation event and when I add custom code in the webhook, it gives multiple duplicate entries of a same order . But in general webhook works only single time on any event. And in my case I received  multiple entries of a same order. 

 **Please check the screenshot :- https://prnt.sc/OUyJHP42X4Wc** 

 Kindly give me your reference on the same asap..

Thanks

**Comment by a-akashsoni** [comment]
@Kyon147 Kindly check the above issue and give your reference on the same asap.

Thanks

**Comment by Kyon147** [comment]
Hi @a-akashsoni 

This is not a package but so going to move this to a discussion.

**Comment by Kyon147** [comment]
The webhooks in the package will just accept everything Shopify send over.

Shopify has quite strict timing for when you need to send back a `200` or `500` for if you got the webhook successfully or not - if there is a delay to this response, Shopify will try again, and keep trying with a decaying time.

So you will need to make sure your code is set up to handle these duplicate events from Shopify.

**Reply by sumantaakeans2020** [reply]
Webhook is not receiving after uninstall the app, i am doing in server. so can you please tell what would be issue?
